### ISSUE TITLE
Screen reader doesn't voice value in mat-select when changed with arrow keys

### ISSUE THREAD

**Issue by calebegg** [angular/material2#11243]
When you change the selection in a mat-select using the arrow keys (without opening the actual listbox), the value of the selection changes but is not announced to a screen reader user

Demoable on https://material.angular.io/components/select/overview with Chromevox or JAWS

Specifically:
1. Tab onto a mat-select
2. Press the down arrow

The selection shown changes, change events fire, but nothing is read aloud by the screen reader.
